workflows/prognostic_c48_run/tests/test_radiation.py

```diff
@@ -7,27 +7,40 @@
 import xarray as xr
 import numpy as np
 import pytest
+import dataclasses
 import os
 
 
+@dataclasses.dataclass
+class PythonRadiativeFlux:
```

Comment:
nit: maybe better named `RadiationDiagnostic` since it includes heating rates. Could also have a `is_flux` property.